src/main/java/maksym/fedorenko/bookstore/model/Book.java

```diff
@@ -44,4 +52,15 @@ public class Book {
 
     @Column(nullable = false)
     private boolean isDeleted = false;
+
+    @ToString.Exclude
+    @EqualsAndHashCode.Exclude
+    @ManyToMany
+    @JoinTable(
+            name = "books_categories",
+            joinColumns = @JoinColumn(name = "book_id"),
+            inverseJoinColumns = @JoinColumn(name = "category_id")
+    )
+    @OrderBy("id")
```

Comment:
Why do you need it?

Reply:
I don't know, this annotation order by categories by id and when I receive a response with categories I will see ordered IDs. Otherwise, they will always be ordered differently for each book.